test/index.test.js

```diff
@@ -0,0 +1,97 @@
+describe('Web test', () => {
+  let createServer
+  let server
+  let mockClaimRepository
+
+  beforeAll(async () => {
+    jest.mock('../server/repository/claim-repository')
+    jest.mock('../server/repository/minetype-repository')
+    jest.mock('../server/services/message-service')

```

Comment:
`minetype-repository` and `message-service` are not used in this test suite.

Reply:
The "submit" page calls methods in claim-repository and message-service. If you don't mock them the claim submission will attempt to communicate with the database and fail when it's not available.

Reply:
Oh of course! My mistake